design-system/react/src/styles/GlobalStyles.tsx

```diff
@@ -7,7 +7,7 @@ const customStyles = {
   body: {
     '-webkit-font-smoothing': 'antialiased',
     '-moz-osx-font-smoothing': 'grayscale',
-    margin: '0',
+    margin: '0 auto',
```

Comment:
I don't think this is a good thing, the design system should not have this by default, it should be neutral in terms of this. Ok, we're using it to an app that is aligned on center, it won't always be like this

Reply:
Ok I will do this directly in wallet them